src/pages/swap/index.tsx

```diff
@@ -168,12 +169,13 @@ export const SwapPage = () => {
     // Do not show any error if the user is disconnected
     if (isDisconnected) return;
 
-    if (typeof userInputTokenBalance === 'string') {
-      if (Big(userInputTokenBalance).lt(fromAmount ?? 0)) {
-        trackEvent({ event: 'form_error', error_message: 'insufficient_balance' });
-        return `Insufficient balance. Your balance is ${userInputTokenBalance} ${fromToken?.assetSymbol}.`;
-      }
-    }
+    // TESTING - REMOVE
```

Comment:
@gianfra-t What about this commented section?

Reply:
Thanks for noticing this one, I forgot to remove after testing. Added a quick fix [here](https://github.com/pendulum-chain/vortex/pull/197).